src/auto/tree_row_reference.rs

```diff
@@ -0,0 +1,71 @@
+// This file was generated by gir (4d68d19) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use TreeModel;
+use TreePath;
+use ffi;
+use glib;
+use glib::object::IsA;
+use glib::translate::*;
+
+glib_wrapper! {
+    pub struct TreeRowReference(Boxed<ffi::GtkTreeRowReference>);
+
+    match fn {
+        copy => |ptr| ffi::gtk_tree_row_reference_copy(mut_override(ptr)),
+        free => |ptr| ffi::gtk_tree_row_reference_free(ptr),
+    }
+}
+
+impl TreeRowReference {
+
+    pub fn new<T: IsA<TreeModel>>(model: &T, path: &TreePath) -> TreeRowReference {
+        skip_assert_initialized!();
+        unsafe {
+            from_glib_full(ffi::gtk_tree_row_reference_new(model.to_glib_none().0, mut_override(path.to_glib_none().0)))
+        }
+    }
+
+    pub fn new_proxy<T: IsA<glib::Object>, U: IsA<TreeModel>>(proxy: &T, model: &U, path: &TreePath) -> TreeRowReference {
```

Comment:
See this [comment](https://github.com/gtk-rs/gtk/pull/387#discussion_r85933474).